lspath: add page (#9558)
tldr-pages/tldr: pages/common/lspath.md

# lspath

> > Print the contents of the system PATH variable, with one element per line.

Reviewer: ```suggestion
> CLI app to list the contents of the PATH environment variable, with optional paging.
```
How about this?

> More information: <https://github.com/clementi/lspath>.

> - Print the contents of the system PATH variable, with one element per line:

Reviewer 2: ```suggestion
- Print the current contents of the system PATH variable, with one element per line:
```

Optional fix: `the current` may sound better.

`lspath`

> - Print the contents of the system PATH variable, with one element per line, with the output paged:

Reviewer 2: ```suggestion
- Print the current contents of the system PATH variable, with one element per line, with the output paged:
```

The same issue as above.

> `lspath -p`

Reviewer: ```suggestion
`lspath --page`
```
Small suggestion, we generally prefer long arguments in tldr.
